src/translations/client.csv

```diff
@@ -1345,11 +1345,11 @@ form.section.informant.name,,Informant,Informateur
 form.section.informant.title,,Informant's details,information de l'informateur
 form.section.information.birth.bullet1,,I am going to help you make a declaration of birth.,Je vais vous aider à faire une déclaration de naissance.
 form.section.information.birth.bullet2,,As the legal Informant it is important that all the information provided by you is accurate.,"En tant qu'informateur légal, il est important que toutes les informations que vous fournissez soient exactes."
-form.section.information.birth.bullet3,,Once the declaration is processed you will receive you will receive an email to tell you when to visit the office to collect the certificate - Take your ID with you.,"Une fois la déclaration traitée, vous recevrez un courriel vous indiquant quand vous rendre au bureau pour retirer le certificat - Munissez-vous d'une pièce d'identité."
+form.section.information.birth.bullet3,,Once the declaration is processed you will receive an email to tell you when to visit the office to collect the certificate - Take your ID with you.,"Une fois la déclaration traitée, vous recevrez un courriel vous indiquant quand vous rendre au bureau pour retirer le certificat - Munissez-vous d'une pièce d'identité."
```

Comment:
For these `bulletN` are defined twice. The french translation seems to change but english stays the same. Let me know if it's not intentional

Reply:
by twice you mean the birth & death ones? In farajaland we are using same translations but it could be different for a country

Reply:
Ah, I misread the strings 😬 . Thanks for clarifying